test/integration/build/.terraform.lock.hcl

```diff
@@ -0,0 +1,21 @@
+# This file is maintained automatically by "terraform init".
```

Comment:
I honestly think we should start using this lock file to lock our external dependencies.. 
@clintoncwolfe  what are your thoughts on the same?